composer.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "umotif/livr-extra",
+    "description": "Lightweight validator supporting Language Independent Validation Rules Specification (LIVR)",
+    "type": "library",
+    "keywords": ["validator", "livr"],
+    "homepage": "https://github.com/umotif-public/php-validator-livr-extra",
+    "license": "MIT",
+    "authors": [
+        {
```

Comment:
Replace this section ane make umotif the author. You can use the engineering@umotif.com email address and the company homepage.